src/components/Home/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import NewPlayer from './NewPlayer';
+import {getSum} from "../../utils";
+import {Button, IconButton} from "@material-ui/core";
+import {DeleteForever} from "@material-ui/icons";
+import {IPlayer} from "../../types";
+
+interface IProps {
+	players: Array<IPlayer>;
+	handleAdd: Function;
+	handleDelete: Function;
+	handleReset: Function;
```

Comment:
'Function' types lacks of signature knowledge. It might seem that all that functions are the same - without arguments and with void result, but from code they are not: 
```
handleAdd: (name: string)=>void;
handleDelete: (id: string) => void;
handleReset: ()=>void;
```

Reply:
Fxied